pages/index.tsx

```diff
@@ -1,10 +1,16 @@
 import * as React from 'react'
 import LandingNav from '../components/LandingNav'
+import LandingHero from '../components/LandingHero'
+import LandingWhy from '../components/LandingWhy'
 
 const IndexPage: React.FC = () => {
   return (
     <div>
       <LandingNav />
+      <div className="container text-center">
+        <LandingHero />
+        <LandingWhy />
```

Comment:
why not dump everything into 1 landing page? I don't see a use case for braking anything up. It makes the landing page more complicated imo.

Reply:
`<LandingPage />` Makes sense.